column_transforms/rasgo_drop_columns/rasgo_drop_columns.sql

```diff
@@ -0,0 +1,44 @@
+{#
+Jinja Macro to generate a query that would get all 
+the columns in a table by source_Id or fqtn
+#}
+{%- macro get_source_col_names(source_id=None, source_table_fqtn=None) -%}
+    {%- set database, schema, table = '', '', '' -%}
+    {%- if source_table_fqtn -%}
+        {%- set database, schema, table = source_table_fqtn.split('.') -%}
+    {%- else -%}
+        {%- set database, schema, table = rasgo_source_ref(source_id).split('.') -%}
+    {%- endif -%}
+        SELECT COLUMN_NAME FROM {{ database }}.information_schema.columns
+        WHERE TABLE_CATALOG = '{{ database }}'
+        AND   TABLE_SCHEMA = '{{ schema }}'
+        AND   TABLE_NAME = '{{ table }}'
+{%- endmacro -%}
+
+{% if include_cols and exclude_cols is defined %}
```

Comment:
does this need to be `if include_cols is defined and exclude_cols is defined`?

Reply:
never mind. Looks like `is defined` is just a truthiness check so `if include_cols` == `if include_cols is defined`.